tests/integration/backup_table_tests/test_backup_table_option.py

```diff
@@ -27,10 +27,16 @@ def check_backup_param_template(self, test_table_name, backup_is_expected):
         with open('target/run/test/models/{}.sql'.format(test_table_name), 'r') as ddl_file:
             ddl_statement = ddl_file.readlines()
             self.assertEqual('backup no' not in ' '.join(ddl_statement).lower(), backup_is_expected)
+            if backup_is_expected:
+                distkey_index = ' '.join(ddl_statement).lower().find('distkey')
```

Comment:
Rather than reinvoking this 3, might as well store it in a variable and run the finds on that? Unless there's a specific state element at play, although I don't believe that's the case.